app/helpers/view_components/student/editor.rb

```diff
@@ -0,0 +1,16 @@
+module ViewComponents
+  module Student
+    class Editor < ViewComponent
+      def initialize(endpoint)
+        @endpoint = endpoint
+      end
```

Comment:
Could this be replaced with a Mandate initialize?

Reply:
I also forget this one! Thanks.